examples/test-LwIP-netconn/scpi_server.c

```diff
@@ -152,7 +147,7 @@ scpi_result_t SCPI_Reset(scpi_t * context) {
 }
```

Comment:
In this context, `SCPI_Reset` is the same type of event so it would be also `weak`.